.gasestimates.md

```diff
@@ -4,126 +4,147 @@ Generated via `bash utils/inspect.sh`.
 ---
 
 `forge test --gas-report --no-match-path "test/script/**/*"`
+| PRECOMPILES contract |                 |      |        |      |         |
+|----------------------|-----------------|------|--------|------|---------|
+| Deployment Cost      | Deployment Size |      |        |      |         |
+| 0                    | 0               |      |        |      |         |
+| Function Name        | min             | avg  | median | max  | # calls |
+| ecrecover            | 3000            | 3000 | 3000   | 3000 | 5       |
+
+
 | lib/account-abstraction/contracts/core/EntryPoint.sol:EntryPoint contract |                 |        |        |        |         |
 |---------------------------------------------------------------------------|-----------------|--------|--------|--------|---------|
 | Deployment Cost                                                           | Deployment Size |        |        |        |         |
-| 4128022                                                                   | 20516           |        |        |        |         |
+| 4781904                                                                   | 23781           |        |        |        |         |
```

Comment:
Do you know why these numbers increased?

Reply:
Hmm not sure. That dependency hasn't changed since this file was last generated.